frontend/javascripts/oxalis/view/left-border-tabs/controls_and_rendering_settings_tab.tsx

```diff
@@ -352,6 +356,26 @@ class ControlsAndRenderingSettingsTab extends PureComponent<ControlsAndRendering
               </SwitchSetting>
             </div>
           )}
+          <SwitchSetting
+            label={
+              <Tooltip title={settingsTooltips.antialiasRendering}>
+                {settingsLabels.antialiasRendering}
+              </Tooltip>
+            }
+            value={this.props.userConfiguration.antialiasRendering}
+            onChange={(arg) => {
+              askUserToReload();
+              this.onChangeUser.antialiasRendering(arg);
+            }}
+          >
+            {this.props.userConfiguration.antialiasRendering && (
+              <Tooltip title="Consider disabling antialiasing if you notice degraded rendering performance.">
+                <ExclamationCircleOutlined
+                  style={{ marginLeft: 8, color: "red", verticalAlign: "middle" }}
```

Comment:
I'd go for orange to get the "warning" feel instead of "error".

Reply:
done :)